test/reducers/ticketReducersTest.js

```diff
@@ -95,6 +104,19 @@ describe('Ticket Reducers', function test () {
       expect(Object.values(this.OnReceiveIncidentFromEmpty).filter(function (ticket) { return ticket.originId === 38808134 })[0]).to.exist
       expect(Object.values(this.OnReceiveIncidentFromPopulated).filter(function (ticket) { return ticket.originId === 38808134 })[0]).to.exist
     })
+
+    it('Should replace a given ticket with null values when REMOVE_TICKET is dispatched', function() {
+      const result = map(populatedTicketList, removeTicketFromRecent(38502026))
+      expect(result[38502026]).to.be.null
```

Comment:
This just set the ticket to null, but didn't remove it from the array. If the array is enumerated, it will still contain the undesired key.

Reply:
Intentional, as removal of the key entirely is not tracked in redux-persist.